docs/src/prototypes/VirtualizedTable/VirtualizedTables.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  Accordion,
+  gridCellBehavior,
+  gridHeaderCellBehavior,
+  gridNestedBehavior,
+  gridRowBehavior,
+  Table,
+} from '@fluentui/react'
+import * as React from 'react'
+import { AutoSizer, List as ReactVirtualizedList, WindowScroller } from 'react-virtualized'
+import getItems from './itemsGenerator'
+
+const scrollbarOffset = 10
+function VirtualizedTablesPrototype() {
+  const [ref, setRef] = React.useState(null)
+
+  const tables = [
+    {
+      key: 'table1',
+      title: <div>Table one</div>,
+      content: <VirtualizedTable scrollElementRef={ref} />,
+    },
+    {
+      key: 'table2',
+      title: <div>Custom table title</div>,
+      content: <VirtualizedTable scrollElementRef={ref} />,
+    },
+  ]
+
+  return (
+    <div id="scrollParent" style={{ height: '700px', overflowY: 'auto' }} ref={setRef}>
+      {ref != null ? <Accordion panels={tables} /> : null}
+    </div>
+  )
+}
+
+interface VirtualizedTableProps {
+  scrollElementRef: React.RefObject<HTMLDivElement>
+}
+
+function VirtualizedTable(props: VirtualizedTableProps) {
+  const { header, rows } = getItems(20, 50)
+  const renderedItems = [header, ...rows]
+
+  const accessibilityListProperties = {
+    'aria-label': '',
+    'aria-readonly': undefined,
+    containerRole: 'presentation',
+    role: 'presentation',
+    tabIndex: null,
+  }
+
+  const rowGetter = ({ index }) => {
+    return renderedItems[index]
+  }
+
+  const rowRenderer = ({ index, style }) => {
+    const row = renderedItems[index]
+    const topOffset = `${style.top}px`
+    const leftOffset = `${style.left}px`
+    const height = `${style.height}px`
+    const header = row.key === 'header'
+    return (
+      <Table.Row
+        design={{
+          top: topOffset,
+          left: leftOffset,
+          width: style.width,
+          height,
+          position: style.position,
+        }}
+        key={row.key}
+        accessibility={gridRowBehavior}
```

Comment:
I have shortly tested the keyboard nav and it does not seem to work correctly - the first table works ok though

Reply:
Sure, will take a look

Reply:
Resolved with help from @kolaps33